src/custom-vocabularies-client.ts

```diff
@@ -0,0 +1,71 @@
+import { ApiRequestHandler } from './api-request-handler';
+import { CustomVocabulary } from './models/CustomVocabulary';
+import { CustomVocabularyOptions } from './models/CustomVocabularyOptions';
+import { RevAiApiCustomVocabulary } from './models/RevAiApiCustomVocabulary';
+
+/**
+* Client to submit and retreive status of custom vocabularies
+* from the rev.ai api. Check rev.ai/docs for more information.
+*/
+export class RevAiCustomVocabulariesClient {
+    apiHandler: ApiRequestHandler;
+
+    /**
+    * @param accessToken Access token used to authenticate API requests
+    * @param version (optional) version of the API to be used
+    */
+    constructor(accessToken: string, version: string = 'v1') {
+        this.apiHandler = new ApiRequestHandler(
+            `https://api.rev.ai/speechtotext/${version}/vocabularies`,
+            accessToken
+        );
+    }
+
+    /**
+    * Submit custom vocabularies to be built. This is primarily
+    * useful for using the custom vocabulary with streaming jobs.
+    * @param customVocabularies array of CustomVocabulary objects.
+    *                           For more information visit rev.ai/docs
+    * @param callbackUrl optional string url to be called when custom
```

Comment:
To be consistent with our param doc formatting:
```suggestion
    * @param callbackUrl (optional) string url to be called when custom
```